### ISSUE TITLE
Add a summary of the specific gameplay differences

### ISSUE THREAD

**Issue by digi-brain** [GameBrains/er-core#51]
At present, the project offers a general overview of the it's purpose and differences from the original _Epic 40,000_ texts, at [About this publication](https://epicremastered.com/core/about-this-publication.html) — but no single, specific list of actual changes to gameplay. 

Whilst we do include footnotes to identify and explain the differences where they occur, it'd still be helpful to summarise these in one place, to list to expand on the general explanation — even though actual gameplay differences are few and modest.

We discussed this a little in [a thread in the Facebook group](https://www.facebook.com/groups/1952676958177757/posts/7521010191344378), which prompted me to log this ticket. As a starting point, refer to the list that Ian created off the top of his head in that thread:

> - Anti Tank changed to Macro Weapon. This is just a cosmetic change with no in-game difference.
> - Cavalry ability split into the movement and unit type part, and the +1 Armour part. This has little effect on army lists as most of the existing Cavalry units now have both abilities. I think it’s just Fleshhounds where I removed the +1 Armour part.
> - Infiltrators<sup>(1)</sup> that are 30-60 cm from their HQ are now just standard parts of the detachment in assaults and for shooting etc. This is largely because we couldn’t work out how the original rules were supposed to work, even with some assistance from Andy Chambers!
> - I’m not sure we got all parts of the Flyer rules the same as the original, again due to the original text not being quite complete.
> - The Speed attribute split into two values for most units, as it’s easier to read for Cavalry etc.
> - Several new abilities added, mostly for new factions that didn’t exist at the time.

<sup>(1)</sup> Note that there may be more to come with the _Infiltrators_ ability rules, as per #50.

Many of those aren't really gameplay differences, but might still be helpful to list I guess. Others of the top of my head include:

- Tweaks we made to the transport rules to even out issues that can occur with the various different base sizes and shapes that players use with their miniatures.

**Comment by IJW-Wartrader** [comment]
This sounds like a great idea.

**Comment by digi-brain** [comment]
Of course, given that we've been at pains to mark up meaningful differences with footnotes, a quick and dirty way for people to find these to review for themselves is:

- Use the project's search tool with a query like: `+difference from +Epic +40,000`
- Use the Google search tool with a query like: `site:epicremastered.com "Difference from Epic 40,000:"`

**Comment by athas** [comment]
One thing I'm wondering about when I'm comparing the Epic Remastered army lists with the original army lists is whether it is a specific goal to have rules for all Legions Imperialis models. It seems some of them have been added, but not all. I think that would be very helpful, as those models are easily accessible (and pretty good).

**Comment by digi-brain** [comment]
> One thing I'm wondering about when I'm comparing the Epic Remastered army lists with the original army lists is whether it is a specific goal to have rules for all Legions Imperialis models. It seems some of them have been added, but not all. I think that would be very helpful, as those models are easily accessible (and pretty good).

Hi @athas! I think that's a bit off-topic for this issue ticket. Here we're thinking about a summary of differences in gameplay that we've created. That is, specific differences in how we interpret rules, and new clauses or rules that we've added — not differences in army lists/detachment lists/unit data sheets.

Beyond that, I'm not quite clear exactly what it is that you're asking. Keep in mind that Legions Imperialis is set in the Horus Heresy, '30k' — whereas Epic 40,000 (and thus Epic Remastered) is set in the 41st millennium ('40k'), as its name indicates. Some units exist in both eras, some don't.

The army lists are primarily driven by @IJW-Wartrader (with some discussion and feedback from others in the community, especially the Facebook group). I don't know if he intends to also create '30k' army lists at any point, with inspiration from Legions Imperialis. There's no reason why anyone else couldn't have a go at that if they want to.
